### ISSUE TITLE
keras ordinal loss

### ISSUE THREAD

**Issue by LucaSCostanzo** [keras-team/keras#21073]
Hi, 
could anyone please give me an insight about what the ordinal loss defined in keras represents?
https://www.tensorflow.org/ranking/api_docs/python/tfr/keras/losses/OrdinalLoss
I'm not able to give an interpretation to that definition, as I don't understand for instance what is I_yi > j.
In the documentation there is no citation of any artiecle.
However, I'm looking for an objective fuction for image classification with ordinal categories, a sort of image ranking loss.
Thanks

**Comment by JyotinderSingh** [comment]
This code is not maintained by the Keras team, it resides under [github.com/tensorflow/ranking](https://github.com/tensorflow/ranking). However, tensorflow/ranking doesn't seem to be under active development.
